packages/visualisation/src/components/TimeProgressBar/index.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+  position: absolute;
+  bottom: 2.5rem;
+  height: 50px;
+  display: flex;
+  align-items: center;
+  left: 15rem;
+  right: 15rem;
+`
+
+const Container = styled.div`
+  height: 8px;
+  width: 100%;
+  zindex: 1000;
+  background-color: #c2c2c1;
+  height: 2px;
+
+  ${Wrapper}:hover & {
+    height: 8px;
+  }
+`
+
+const Filler = styled.div`
+  height: '100%';
+  width: ${({ completed }) => `${completed}%`};
+  background-color: #13c57b;
+  transition: 'width 1s ease-in-out';
+  height: 2px;
+
+  ${Wrapper}:hover & {
+    height: 8px;
+  }
+`
+
+const Circle = styled.div`
+  height: 10px;
+  width: 10px;
+  background-color: #13c57b;
+  border-radius: 50%;
+  position: absolute;
+  top: 1.25rem;
+  left: ${({ completed }) => `${completed - 0.3}%`};
```

Comment:
This `0.3%` and the below `1.2%` might be good on a small screen, but on a larger the circle appears off center when enlarged for hover. Maybe use something like `calc(${completed}% - 5px)` or something similar?